### DISCUSSION TITLE
Please add Firefox to F-Droid

### DISCUSSION THREAD

**Discussion by Lhy220402** [mozilla-mobile/fenix#28743] [Ideas and Feature Requests]
GooglePlay is not accessible in China and downloading on github is slow. Can firefox be added to F-Droid?Please!
中国无法访问GooglePlay，在github上下载又很慢，可以把firefox添加至F-Droid吗？
                                                                                                                 --A Firefox user in China

**Comment by cadeyrn** [comment]
Firefox is already on F-Droid, it's called Fennec there.

**Reply by Lhy220402** [reply]
thankyou！

**Reply by thomas-rhong** [reply]
Fennec is somehow worse than Firefox. For example, Fennec does not provide Baidu as a search engine which means you have to add it manually; thus you cannot use "Search Suggestions". 
(In many ways Baidu is better than any other engines while search via Chinese so Baidu is unreplaceable. )

**Comment by thomas-rhong** [comment]
Use Watt Toolkit. For Chinese to use many websites it's useful. 
（这原来叫Steam++，用来给Steam加速，同时也有Github加速的功能，比较方便。）
